lib/helper/rules/PricerRule.js

```diff
@@ -0,0 +1,183 @@
+
+    return txReceipt;
+  }
+
+  /**
+   * Sets an acceptance margin for the base currency price per pay
+   *         currency. From should be worker only.
+   *
+   * @param payCurrencyCode QuoteCurrency code. e.g. ETH, BTC.
+   * @param acceptanceMargin Acceptance margin for the base currency price per pay currency.
+   * @param txOptions Tx options.
+   * @returns {Promise<*>}
+   */
+  async setAcceptanceMargin(payCurrencyCode, acceptanceMargin, txOptions) {
+    const oThis = this;
+    let txObject = oThis._setAcceptanceMarginRawTx(payCurrencyCode, acceptanceMargin);
+
+    let txReceipt;
+    txReceipt = await new TxSender(txObject, oThis.auxiliaryWeb3, txOptions).execute();
+
+    return txReceipt;
+  }
+
+  /**
+   * Removes an acceptance margin of the base currency price in the
+   *         specified pay currency. From should be worker only.
+   *
+   * @param payCurrencyCode QuoteCurrency code. e.g. ETH, BTC.
+   * @param txOptions Tx options.
+   * @returns {Promise<*>}
+   */
+  async removeAcceptanceMargin(payCurrencyCode, txOptions) {
+    const oThis = this;
+    let txObject = oThis._removeAcceptanceMarginRawTx(payCurrencyCode);
+
+    let txReceipt;
+    txReceipt = await new TxSender(txObject, oThis.auxiliaryWeb3, txOptions).execute();
+
+    return txReceipt;
+  }
+
+  /**
+   * Adds a new price oracle.
+   *
+   * @param priceOracleAddress PriceOracle contract address.
+   * @returns {Promise<*>} Promise object.
+   * @private
+   */
+  _addPriceOracleRawTx(priceOracleAddress) {
+    const oThis = this;
+
+    return oThis._pricerRuleContractInstance().methods.addPriceOracle(priceOracleAddress);
+  }
+
+  /**
+   * Removes the price oracle for the specified pay currency code.
+   *
+   * @param payCurrencyCode QuoteCurrency code. e.g. ETH, BTC.
+   * @returns {Promise<*>} Promise object.
+   * @private
+   */
+  _removePriceOracleRawTx(payCurrencyCode) {
+    const oThis = this;
+
+    const bytesPayCurrencyCode = oThis.auxiliaryWeb3.utils.stringToHex(payCurrencyCode.toString());
+    return oThis._pricerRuleContractInstance().methods.removePriceOracle(bytesPayCurrencyCode);
+  }
+
+  /**
+   * Sets an acceptance margin for the base currency price per pay
+   *      currency. From should be worker only.
+   * @param payCurrencyCode QuoteCurrency code. e.g. ETH, BTC.
+   * @param acceptanceMargin Acceptance margin for the base currency price per pay currency.
+   * @returns {*}
+   * @private
+   */
+  _setAcceptanceMarginRawTx(payCurrencyCode, acceptanceMargin) {
+    const oThis = this;
+
+    const bytesPayCurrencyCode = oThis.auxiliaryWeb3.utils.stringToHex(payCurrencyCode.toString());
+    return oThis._pricerRuleContractInstance().methods.setAcceptedMargin(bytesPayCurrencyCode, acceptanceMargin);
+  }
+
+  /**
+   * Removes an acceptance margin of the base currency price in the
+   *         specified pay currency. From should be worker only.
```

Comment:
Suggestion :-
`From address should be only worker address.`
Please update in other methods also.